### PR TITLE
make it possible to configure scripts with reversed exit codes

### PR DESCRIPTION
**Pull request by zorafuchs**
- add attributes to ScriptConfig
	- add new factory for ScriptConfig with reversed_exit that is true
	- make ServantThread set the outputs/error-outputs/exit codes the right way

### CODE REVIEW

File: lib/renuo/bin-check/servant_thread.rb [hunk 1]
```diff
@@ -20,8 +20,18 @@ def run_with_cache
     end
 
     def run_command
+      @script.reversed_exit? ? run_reversed : run_normally
+    end
+
+    def run_normally
+      Open3.popen3(@script.script_command) do |_stdin, stdout, stderr, wait_thr|
```
Comment (lukaselmer) [inline lib/renuo/bin-check/servant_thread.rb]: i would refactor this, so you only need the complicated Open3.popen3 once.

File: spec/factories/script_config.rb [hunk 2]
```diff
@@ -4,6 +4,7 @@
     script_command 'cool command'
     script_name 'script_name'
     script_files %w(cool_file1 cool_file1)
+    script_reversed_exit false
```
Comment (lukaselmer) [inline spec/factories/script_config.rb]: is this needed?

**Comment by lukaselmer** [comment]
lgtm!

**Comment by sihu** [comment]
LGTM!

**Pull request merged**